lib/transactions/models.py

```diff
@@ -92,21 +94,23 @@ def is_refunded(self):
             status=constants.STATUS_COMPLETED).exists()
 
     def for_log(self):
-        return (
-            'v5',  # Version.
-            self.uuid,
-            self.created.isoformat(),
-            self.modified.isoformat(),
-            self.amount,
-            self.currency,
-            self.status,
-            self.status_reason,
-            self.buyer.uuid if self.buyer else None,
-            self.seller_product.seller.uuid if self.seller_product else None,
-            self.source,
-            self.carrier,
-            self.region,
-            self.provider)
+        return OrderedDict((
+            ('version', 'v5'),  # Version.
+            ('uuid', self.uuid),
+            ('created', self.created.isoformat()),
+            ('modified', self.modified.isoformat()),
+            ('amount', self.amount),
+            ('currency', self.currency),
+            ('status', self.status),
+            ('reason', self.status_reason),
+            ('buyer', self.buyer.uuid if self.buyer else None),
+            ('seller', (self.seller_product.seller.uuid
+                        if self.seller_product else None)),
+            ('source', self.source),
+            ('carrier', self.carrier),
+            ('region', self.region),
+            ('provider', self.provider)
+        ))
```

Comment:
Bad news, reason was added last month:

https://github.com/mozilla/solitude/commit/705ac69dd03638e081ee87b8b1ccc63cf8f9f0aa#diff-3d2b1aa083001b6a0456c9031c1ee15cR96

And I bumped the version then.